### PR TITLE
Add 13 python packages

### PR DESCRIPTION
**Pull request by mfrw**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Add 13 pythonn-* extended packages.

Packages added:
- google-api-python-client
- python-dbusmock
- python-ddt
- python-fasteners
- python-httplib2
- python-memcached
- python-msal
- python-msrestazure
- python-paramiko
- python-pytest-relaxed
- python-testpath
- python-waitress
- python-yubico

###### Change Log  

- google-api-python-client: add package from f36 (automated) [fixed mfrw]
- google-api-python-client: cleanup and add CL
- google-api-python-client: move summary to the top
- python-fasteners: add package from f36 (automated) [fixed mfrw]
- python-fasteners: add changelog
- python-msal: add package from f36 (automated) [fixed mfrw]
- python-msal: add changelog
- python-msrestazure: add package from f36 (automated) [fixed mfrw]
- pythonn-msrestazure: add changelog
- python-pytest-relaxed: add package from f36 (automated) [fixed mfrw]
- python-pytest-relaxed: build with version 2.0.0
- python-pytest-relaxed: add changelog
- python-dbusmock: add package from f36 (automated)
- python-dbusmock: add changelog
- python-ddt: add package from f36 (automated)
- python-ddt: add changelog
- python-httplib2: add package from f36 (automated)
- python-httplib2: add changelog
- python-memcached: add package from f36 (automated)
- python-memcached: add changelog
- python-memcached: fix source0 name
- python-paramiko: add package from f36 (automated)
- python-paramiko: drop docs
- python-paramiko: add changelog
- python-testpath: add package from f36 (automated)
- python-testpath: add changelog
- python-waitress: add package from f36 (automated)
- python-waitress: add changelog
- python-waitress: fix source URL
- python-yubico: add package from f36 (automated)
- python-yubico: add changelog
- python-*: remove checked-in sources
- python-*: license: add entries
- cgmanifest: sort entries
- python-*: cgmanifest: add entries

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- NA

###### Test Methodology

- Local Build

Buddy Build: 
- [AMD](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=321864&view=logs&j=7909073b-e471-53c1-2c44-627e2efc0564&t=a3bfadd1-83e5-5c19-e443-1c88a6f53803&l=1704)
- [ARM](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=321864&view=logs&j=21e55549-fa44-57d8-4936-64132e3ceec0&t=0085d9b2-2408-5abe-87ce-b702734e35d8&l=1704)

### CODE REVIEW

File: SPECS-EXTENDED/python-msal/python-msal.spec [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Version:        1.18.0~b1
+Name:           python-%{srcname}
+Release:        2%{?dist}
+Summary:        Microsoft Authentication Library (MSAL) for Python
+
+License:        MIT
+URL:            %forgeurl
+Source0:        %forgesource
```
Comment (kanikanema) [inline SPECS-EXTENDED/python-msal/python-msal.spec:19]: Append #/%{name}-%{version}.tar.gz. Please check the other spec files for this as well.

File: SPECS-EXTENDED/python-msrestazure/python-msrestazure.spec [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+%build
+%pyproject_wheel
+
+
+%install
+%pyproject_install
+%pyproject_save_files %{srcname}
+
+
+%if %{with tests}
```
Comment (Sumynwa) [inline SPECS-EXTENDED/python-msrestazure/python-msrestazure.spec:64]: Remove if/else for check section

**Review by kanikanema — APPROVED** [review]
LGTM.

File: SPECS-EXTENDED/python-yubico/python-yubico.spec [hunk 3]
```diff
@@ -0,0 +1,173 @@
+%{?python_enable_dependency_generator}
```
Comment (Sumynwa) [inline SPECS-EXTENDED/python-yubico/python-yubico.spec:1]: We can remove this macro not present in mariner

File: SPECS-EXTENDED/python-testpath/python-testpath.spec [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+Source0:        %pypi_source
+BuildArch:      noarch
+
+BuildRequires:  python3-pip
+BuildRequires:  python3-devel
+BuildRequires:  python3-flit
+BuildRequires:  python3-sphinx
+BuildRequires:  python3-sphinx_rtd_theme
+
+# Tests:
```
Comment (Sumynwa) [inline SPECS-EXTENDED/python-testpath/python-testpath.spec:20]: Use with_check macro to pull in check dependencies.

File: SPECS-EXTENDED/python-ddt/python-ddt.spec [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Name:           python-%{pkgname}
+Version:        1.4.2
+Release:        6%{?dist}
+Summary:        Python library to multiply test cases
+License:        MIT
+URL:            https://github.com/datadriventests/ddt
+Source0:        %pypi_source
```
[the remark this answers is not in the document]
Comment (mfrw) [reply SPECS-EXTENDED/python-ddt/python-ddt.spec:25]: This macro resolves it properly :)

File: SPECS-EXTENDED/python-ddt/python-ddt.spec [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+
+
+%description %{common_description}
+
+
+%package -n python3-%{pkgname}
+Summary:        %{summary}
+BuildRequires:  python3-devel
+BuildRequires:  %{py3_dist setuptools}
+%if %{with tests}
```
Comment (Sumynwa) [inline SPECS-EXTENDED/python-ddt/python-ddt.spec:36]: Kindly use with_check macro instead

File: SPECS-EXTENDED/python-fasteners/python-fasteners.spec [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Name:           python-fasteners
+Version:        0.18
+Release:        2%{?dist}
+Summary:        A python package that provides useful locks
+License:        Apache-2.0
+URL:            https://github.com/harlowja/fasteners
+# We need to use the GitHub archive instead of the PyPI sdist to get tests.
+Source0:        %{url}/archive/%{version}/fasteners-%{version}.tar.gz
```
[the remark this answers is not in the document]
Comment (mfrw) [reply SPECS-EXTENDED/python-fasteners/python-fasteners.spec]: Done

**Review by Sumynwa — APPROVED** [review]
Some nitpick w.r.t using with_check macro for pulling check section dependencies.

**Comment by mfrw** [comment]
Thank you @kanikanema & @Sumynwa for the review.

**Pull request merged**